core/src/main/scripts/importer/validateData.py

```diff
@@ -3353,7 +3354,7 @@ def process_metadata_files(directory, portal_instance, logger, relaxed_mode, str
 
     # get filenames for all meta files in the directory
     filenames = [os.path.join(directory, f) for
-                 f in os.listdir(directory) if
+                 f in sorted(os.listdir(directory)) if
```

Comment:
why this change?

Reply:
To be able to pass the tests. `os.listdir` returns a list in arbitrary order (https://docs.python.org/3.5/library/os.html#os.listdir).